apis/capabilities/v1beta1/product_types.go

```diff
@@ -59,6 +60,19 @@ const (
 	ProductFailedConditionType common.ConditionType = "Failed"
 )
 
+var (
+	// ApicastPolicy refers to the main functionality of APIcast to work with the 3scale API manager
+	// Needs to exist in the policy chain
+	ApicastPolicy = PolicyConfig{
```

Comment:
Is this a global public variable for some reason? Do other packages need to access this?